lib/poclu/misc.c

```diff
@@ -198,6 +198,68 @@ poclu_write_file (const char *filemane, char *content, size_t size)
   return 0;
 }
 
+cl_int
+poclu_show_program_build_log (cl_program program)
+{
+  cl_int err;
+  cl_uint num_devices;
+  cl_device_id *devices;
+
+  err = clGetProgramInfo(program, CL_PROGRAM_NUM_DEVICES, sizeof(num_devices), &num_devices, NULL);
```

Comment:
These rows look long, can you run the clang formatter script on this commit?

Reply:
done